lib/interfaces/common.js

```diff
@@ -50,6 +50,22 @@ module.exports = function (suites, context, mocha) {
       };
     },
 
+    /**
+     * Check for nested test registration.
+     *
+     * @param {Suite} suite The suite to which a test is being added.
+     * @param {string} title The title of the test being added.
+     */
+    checkForNestedTest: function checkForNestedTests(suite, title) {
+      // Check for nested test registration
```

Comment:
[Naming] The names didn't match between the export and the function declaration. Plus the comment said roughly the same thing a third time.

```suggestion
    checkForNestedTest: function checkForNestedTest(suite, title) {
```